packages/devtools/lib/src/timeline/timeline_protocol.dart

```diff
@@ -586,75 +590,73 @@ class TimelineFrame {
   bool get isReadyForTimeline {
     return uiEventFlow != null &&
         gpuEventFlow != null &&
-        _pipelineItemStartTime != null &&
-        _pipelineItemEndTime != null;
+        pipelineItemStartTime != null &&
```

Comment:
any time you have startTime and endTime in a variable name it should be TimeRange object.

Reply:
done.